### PR TITLE
Maid outfit for janidrobe

### PR DESCRIPTION
**Pull request by Emisse**
changelog
-add maid outfit to emagged janidrobe
-add clown BLING shoes to emagged theater drobe (they glow slightly) 

look me in the eyes and tell me you dont want this
image
image

### CODE REVIEW

**Comment by vulppine** [comment]
just add an entire maid uniform and stuff it into contraband 😎

**Comment by Emisse** [comment]
based idea

**Comment by Snowyypanda** [comment]
the tg maid outfit looks much better imo

**Comment by Elijahrane** [comment]
> the tg maid outfit looks much better imo

this literally is the /tg/ janitor maid outfit

File: Resources/Prototypes/Entities/Clothing/Shoes/specific.yml [hunk 1]
```diff
@@ -23,6 +23,26 @@
...
+  - type: Sprite
+    sprite: Clothing/Shoes/Specific/bling.rsi
+  - type: Clothing
+    sprite: Clothing/Shoes/Specific/bling.rsi
+  - type: FootstepModifier
+    footstepSoundCollection:
+      collection: footstep_clown
+  - type: PointLight
+    netsync: false
+    enabled: false
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Clothing/Shoes/specific.yml:41]: not enabled?

**Review by mirrorcult — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Clothing/Shoes/specific.yml (hunk 1 above)
Comment (Emisse) [reply Resources/Prototypes/Entities/Clothing/Shoes/specific.yml]: it sit absoneobfeon efroer renanbled

**Pull request merged**